inc/saitypes.h

```diff
@@ -1619,6 +1619,19 @@ typedef enum _sai_object_stage_t
 
 } sai_object_stage_t;
 
+typedef enum _sai_health_data_type_t
+{
+    /** General health data type */
+    SAI_HEALTH_DATA_TYPE_GENERAL
+} sai_health_data_type_t;
+
+typedef struct _sai_switch_health_data_t
+{
+    /** Type of switch health data */
+    sai_health_data_type_t data_type;
```

Comment:
since this struct contains only one entry, is it not better to use data_type directly in notification ?

Reply:
since we want to make structs constant then this would not expand any more